src/librawspeed/decoders/RawDecoder.cpp

```diff
@@ -125,8 +125,6 @@ void RawDecoder::decodeUncompressed(TiffIFD *rawIFD, BitOrder order) {
 
 bool RawDecoder::checkCameraSupported(CameraMetaData *meta, string make,
                                       string model, const string &mode) {
-  TrimSpaces(make);
```

Comment:
> E.g. see aridecoder, it calls with some string

But if a programmer used that function and passed it an invalid compile-time string (including a space at the end), would the right thing to do in that case not be to fix that string where it is specified instead of fixing it during runtime?

Is seems quite obvious to me that this particular trim here _is_ the 'default' trim someone came up with to deal with the problem of excessive spaces in tiff entries on a more or less central position. I simply put it, where it belongs.